packages/@aws-cdk/aws-sqs/lib/queue.ts

```diff
@@ -87,7 +87,7 @@ export interface QueueProps {
    * Be aware that encryption is not available in all regions, please see the docs
    * for current availability details.
    *
-   * @default Unencrypted
+   * @default SQS_MANAGED (SSE-SQS) for newly created queues, otherwise unencrypted
```

Comment:
```suggestion
   * @default SQS_MANAGED (SSE-SQS)
```

I wouldn't differentiate between newly created or not here. How do I know whether _my_ queue is "newly" created or not?

Reply:
Yes, you don't know when the queue was created and whether it uses the old or the new default value. I changed the description as proposed.